### PR TITLE
Update Main.java

### PR DESCRIPTION
**Pull request by Yorm-sk**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello ga");
+        System.out.println("Hello Den");
```
Comment (InPhysicsWeTrust) [inline src/Main.java:3]: bbb

Comment (InPhysicsWeTrust) [reply src/Main.java:3]: fock

Comment (Yorm-sk) [reply src/Main.java:3]: Fock you

Comment (InPhysicsWeTrust) [reply src/Main.java:3]: fock u too

**Pull request merged**